CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+## v1.1.0
+### 2024-09-10
+
+This version of HOSS provides support for products that do not comply with CF standards like SMAP L3.
```

Comment:
This change log is much better, thanks, but this first sentence still doesn't really tell someone what this change supports. These changes help with non-CF-Convention compliant collections, but how are those collections non-CF-Convention  compliant?

Reply:
done - https://github.com/nasa/harmony-opendap-subsetter/pull/15/commits/2296a35f4a0a94038839e205a434051c2590ca21

Reply:
I think this now, in the aggregate, describes the changes. Generally, though, I think it is better to describe what the functionality does with minimal references to the products. The description still leans a little heavily on SMAP L3 in particular.

It's a minor thing, and I'm not going to dig in any further over it, but instead of:

> This version of HOSS provides support for products that do not comply with CF standards like SMAP L3.
The SMAP L3 products do not have grid mapping variable to provide the projection information.
They also do not have x-y dimension scales for the projected grid.

Probably better would have been:

> This version of HOSS provides support for gridded products that do not contain CF-Convention compliant grid mapping variables and 1-D dimension variables, such as SMAP L3.